.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-2.2
```

Comment:
@mvz I'm aware you said this caused an issue on your rbenv, but we really need to be running 2.2 locally at some point, so I'd argue this is worth keeping in, whilst we're supporting this low version.

On a side-note, are we planning on upping what versions we support?

Reply:
> we really need to be running 2.2 locally at some point

Can you explain why? I think running 2.2 in CI is enough, while using whichever later version is convenient for local development.

Reply:
Whilst ruby doesn't adhere strictly to semver, we should be developing things which work against 2.2 (Using the assumption that if they work on 2.2, they should work on 2.2+)

Extending that logic across the local/CI dev tree, then suggests we should be developing against 2.2. It is done in the cucumber-ruby repo, which is the closest mirror to this.

Reply:
I don't see `.ruby-version` in the `cucumber-ruby` repository, and it is even explicitely ignored in their `.gitignore`: https://github.com/cucumber/cucumber-ruby/blob/master/.gitignore#L29. Maybe that makes sense to do here as well?

Reply:
Evidently this has changed recently. Mybad :(

If I add it to the gitignore is that ok?

Reply:
No problem. Adding to gitignore is fine I think.